aeson.cabal

```diff
@@ -175,7 +175,7 @@ library
     hs-source-dirs: ffi
     other-modules: Data.Aeson.Parser.UnescapeFFI
 
-test-suite tests
+test-suite aeson-tests
```

Comment:
I think this name is mentioned in CONTRIBUTING.md

Reply:
Updated.